deploy/helm/elastic-agent/templates/agent/service-account.yaml

```diff
@@ -11,9 +12,10 @@ metadata:
     {{- with ($presetVal).labels -}}
     {{ toYaml . | nindent 4 }}
     {{- end }}
-  {{- with ($presetVal).annotations }}
+  {{- with ($presetVal).serviceAccount.annotations }}
```

Comment:
Shouldn't this be both the preset-wide annotations *and* the ServiceAccount-specific ones?